grazer/src/main/java/edu/asu/diging/grazer/web/LoginController.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.grazer.web;
+
+import java.security.Principal;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+public class LoginController {
+
```

Comment:
Most of the methods in this class are present in the HomeController.java.  Its best not to duplicate the functionality across multiple classes.